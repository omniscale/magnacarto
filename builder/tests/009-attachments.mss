#class::foo {
    [type='foo']{
        line-width: 1;
    }
}

#class::bar[zoom=1] {
    [type='foo']{
        line-width: 1;
    }
    [type='baz']{
        line-width: 2;
    }
}

#class::foo {
    [type='foo']{
        line-width: 1;
    }
}
