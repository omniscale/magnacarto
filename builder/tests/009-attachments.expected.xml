<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="class-foo" filter-mode="first">
    <Rule>
      <Filter>([type] = &#39;foo&#39;)</Filter>
      <LineSymbolizer stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Style name="class-bar" filter-mode="first">
    <Rule>
      <!--Zoom{=1}-->
      <MaxScaleDenominator>500000000</MaxScaleDenominator>
      <MinScaleDenominator>200000000</MinScaleDenominator>
      <Filter>([type] = &#39;baz&#39;)</Filter>
      <LineSymbolizer stroke-width="2"></LineSymbolizer>
    </Rule>
    <Rule>
      <!--Zoom{=1}-->
      <MaxScaleDenominator>500000000</MaxScaleDenominator>
      <MinScaleDenominator>200000000</MinScaleDenominator>
      <Filter>([type] = &#39;foo&#39;)</Filter>
      <LineSymbolizer stroke-width="1"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="class" srs="" status="off">
    <StyleName>class-foo</StyleName>
    <StyleName>class-bar</StyleName>
  </Layer>
</Map>